README.md

```diff
@@ -1,16 +1,20 @@
 # `nixfmt`
 
-`nixfmt` is a formatter for Nix code, intended to easily apply a uniform style.
-
-You are encouraged to test this out on your code and submit any undesirable formatting you find as an issue
+`nixfmt` is the official formatter for Nix language code, intended to easily apply a uniform style.
 
 ![Build Status](https://github.com/NixOS/nixfmt/actions/workflows/main.yml/badge.svg?branch=master)
 
 ## State
 
-`nixfmt` will form the basis for the initial official standard Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+`nixfmt` was originally developed by [Serokell](https://github.com/serokell).
+It was used as the basis for the official standardised Nix formatter, as established by [RFC 166](https://github.com/NixOS/rfcs/pull/166).
+
+The official standard differs considerably from the original implementation.
+Be aware of this if you track the [`master`](https://github.com/NixOS/nixfmt/tree/master) branch.
+Until the first new release, `nixfmt` should be considered **very unstable**.
```

Comment:
(not your fault, feel free to ignore):

I don't think it would be clear to readers what "the first new release" means here. Should we link to https://github.com/NixOS/nixfmt/issues/272 instead?